action.php

```diff
@@ -478,9 +478,16 @@ protected function sendPDFFile($cachefile) {
         header('Cache-Control: must-revalidate, no-transform, post-check=0, pre-check=0');
         header('Pragma: public');
         http_conditionalRequest(filemtime($cachefile));
+        global $INPUT;
+        if ($INPUT->has('outputTarget')) {
+            $outputTarget = $INPUT->str('outputTarget');
+        }
+        if (empty($outputTarget)) {
+            $outputTarget = $this->getConf('output');
+        }
```

Comment:
input->str() can be supplied with a default which is used when the parameter is empty. the makes the above a one liner